docs/topics/azure-stack.md

```diff
@@ -298,7 +298,6 @@ In this section, please follow the example commands to deploy a StatefulSet appl
 DRIVER_VERSION=v1.26.5 # if using k8s v1.26
 DRIVER_VERSION=v1.28.3 # if using k8s v1.27
 DRIVER_VERSION=v1.29.1 # if using k8s >= v1.28
```

Comment:
Please use the latest supported version for CSI Driver for 1.29.*.
Currently it is v1.29.14 .
Please verify that on Azure Stack Hub.